python/dgl/mock_sparse/diag_matrix.py

```diff
@@ -123,6 +123,38 @@ def as_sparse(self) -> SparseMatrix:
         row = col = torch.arange(len(self.val)).to(self.device)
         return create_from_coo(row=row, col=col, val=self.val, shape=self.shape)
 
+    def t(self):
+        """Alias of transpose()
```

Comment:
Use autodoc to link the the function. Or use See Also section. See https://docs.dgl.ai/generated/dgl.DGLGraph.number_of_nodes.html#dgl.DGLGraph.number_of_nodes for an example.

Reply:
done